### PR TITLE
Add exclusive features

### PR DESCRIPTION
**Pull request by fernandog**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -5,14 +5,23 @@
...
+ - Supports TVMaze and TMDB indexers
  - Manual search episodes (you choose what to snatch based on all kind of info: provider, seeds, release group)
  - Manual search for subtitles (useful when subtitle has low score because of typo in filename or alternative title)
  - Uses 'guessit' library to parse and enrich existing info (more precise than regexes)
  - Improved Anime shows support
+ - Faster DailySearcher as we only process new items. Already processed items are discarded.
+ - Option to clean Kodi library when replacing existing files
+ - Better quality explanations and consistent quality code in all threads
+ - See on the fly if your backlogged episodes will be increased/reduced while changing qualities
+ - Remove torrent from Transmission (alpha feature) when ratio reached
```
Comment (p0psicles) [inline readme.md]: Please remove this one. It's not a feature (yet)

Comment (p0psicles) [inline readme.md]: Maybe add, Nightly showupdater updates only the season of the show, which has been updated by the indexer, for tvdb and tmdb.

**Comment by fernandog** [comment]
any missing exclusive features?

File: readme.md [hunk 2]
```diff
@@ -5,14 +5,23 @@
...
+ - Supports TVMaze and TMDB indexers
  - Manual search episodes (you choose what to snatch based on all kind of info: provider, seeds, release group)
  - Manual search for subtitles (useful when subtitle has low score because of typo in filename or alternative title)
  - Uses 'guessit' library to parse and enrich existing info (more precise than regexes)
  - Improved Anime shows support
+ - Faster DailySearcher|Backlog|Find propers as we only process new items. Already processed items are discarded.
+ - Option to clean Kodi library when replacing existing files
+ - Better quality explanations and consistent quality code in all threads
+ - See on the fly if your backlogged episodes will be increased/reduced while changing qualities
+ - Postpone PostProcessing until Medusa downloads wanted subtitle (usefull to only show media if subtitle available)
```
Comment (OmgImAlexis) [inline readme.md]: Add a space between the two words "Post Processing".

Comment (OmgImAlexis) [inline readme.md]: "Useful" has only one l.

File: readme.md [hunk 3]
```diff
@@ -5,14 +5,23 @@
...
+ - Clean any left overs files/folders if media file get deleted
```
Comment (OmgImAlexis) [inline readme.md]: Change to "Clean up any leftover files/folders if media file is deleted"

**Comment by fernandog** [comment]
@OmgImAlexis done

**Pull request merged**